index.html

```diff
@@ -283,6 +283,29 @@ <h5>Languages</h5>
                     </div>
             </div>
         </section>
-   
+   <section class="footer">
+    <div id="ellipse122"><img src="Ellipse122.png" alt=""></div>
+    <div id="rectangle44"><img src="Rectangle44.png" alt=""></div>
+    <div id="ellipse33"><img src="Ellipse33.png" alt=""></div>
+    <div class="contact"><h6>Contact me</h6></div>
+    <div id="content">
+        <p>
+            If you have an application you are interested in developing, 
+            a feature that you need built or a project that needs coding. 
+            I’d love to help with it 
+        </p>
+    </div>
+    <form action="https://formspree.io/f/xjvlqzdr"  method="post">
+        <input type="text" name="user_name" placeholder="name" id="name" maxlength="30" required>
+        <input type="email" name="user_email" placeholder="E-mail" id="email" required>
```

Comment:
- [ ] [OPTIONAL] - I would recommend that you make the placeholder text more descriptive. Having just "name" and "E-mail" would not really help the users of your website know what to put. Just one name or both names? or what format for email? Always design your websites with the assumption that the users will not be computer literate and helping them know exactly what to put will increase your project's user experience.